katacomb/katacomb/conf/parameters/MKAT_wide_UHF.yaml

```diff
@@ -1,4 +1,4 @@
-# Default UHF-band MFImage configuration options for 'online' mode
+# Default UHF-bandconfiguration options for wideband 'online' mode
```

Comment:
Please insert a space between UHF-band and configuration.

Reply:
Done.